cli/src/commonMain/kotlin/ReportsCommand.kt

```diff
@@ -86,7 +85,13 @@ class ReportsCommand : SuspendingCliktCommand(name = "reports") {
 
         outputDir.mkdirs()
 
-        fileNames.forEach { fileName ->
+        val reportNames = fileNames ?: run {
+            val ortRun = client.runs.getOrtRun(resolvedOrtRunId)
+
+            ortRun.jobs.reporter?.reportFilenames
+        } ?: emptyList()
```

Comment:
Nit: You could make it a bit shorter:
```kotlin
val reportNames = fileNames
  ?: client.runs.getOrtRun(resolvedOrtRunId).jobs.reporter?.reportFilenames
  ?: emptyList()
```

Reply:
It becomes yet shorter by using `orEmpty()` 😸

Reply:
Yes, it is shorter.
But IMO the readability gets worse:
```kt
val reportNames = fileNames ?: client.runs.getOrtRun(resolvedOrtRunId).jobs.reporter?.reportFilenames.orEmpty()
```

I would like to keep it as is
(I replaced `?: emptyList()` with `.orEmpty()`)

Alternatively, I would extract the ORT run fallback to a private function.